src/App.tsx

```diff
@@ -11,14 +11,17 @@ export class App extends React.Component<{}, State> {
   };
 
   addOne = () => {
-    // write code here
+    this.setState(state => ({
```

Comment:
Better to explicitly write that you're working with previous state, it can easily confuse
```suggestion
    this.setState(prevState => ({
```